grails-rundeck-data-shared/src/main/groovy/rundeck/data/util/WorkflowStepUtil.groovy

```diff
@@ -0,0 +1,37 @@
+package rundeck.data.util
+
+import org.rundeck.app.data.model.v1.job.workflow.WorkflowStepData
+import org.rundeck.core.execution.BaseCommandExec
+import rundeck.data.constants.WorkflowStepConstants
+
+class WorkflowStepUtil {
+
+    static String summarize(WorkflowStepData step) {
+        if(step.pluginType == WorkflowStepConstants.TYPE_COMMAND & step instanceof BaseCommandExec) {
```

Comment:
The RdWorkflowStep doesn't implement BaseCommandExec, so you can't use that as part of the check.